recipes/libnuma/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+class LibnumaConan(ConanFile):
+    name = "libnuma"
+    description = "NUMA support for Linux."
+    license = "GPL-2.0-or-later"
+    topics = ("conan", "numa")
+    homepage = "https://github.com/numactl/numactl"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    exports_sources = "patches/**"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("{} is only supported on Linux".format(self.name))
```

Comment:
```suggestion
        if not self.settings.os in ["Linux", "FreeBSD"]:
            raise ConanInvalidConfiguration("{} is only supported on Linux/FreeBSD".format(self.name))
```

Reply:
libnuma is not supported on FreeBSD. maybe NUMA as technology is available but `shm.c` fails to compile due to missing `fstat64`, `mmap64`. Reverting.